src/fuzzy.c

```diff
@@ -1,1657 +0,0 @@
-/* Generated by Pyrex 0.9.8.6 on Sat Feb  4 15:03:10 2012 */
```

Comment:
While it's nice to omit the generated code, it's necessary to create sdists that can be installed without Cython, which presumably was a goal at one point. I'm happy to omit this file, but we should be explicit about which installations this might break and how to mitigate the effects.